packages/office-addin-test-server/package.json

```diff
@@ -37,6 +37,7 @@
         "concurrently": "^6.3.0",
         "mocha": "^9.1.1",
         "office-addin-lint": "^2.0.0",
+        "office-addin-usage-data": "^1.4.6",
```

Comment:
Is the cli supposed to be removed?

Reply:
It previous only imported cli for the `logError` function, which is moving packages

Reply:
Yes, but you're keeping the package import for the cli as well as adding the usage-data package that has the methods new location.

Reply:
Yeah, both would work, but we are now deprecating "logError" from the cli package, so this is just us moving all calls from it to usage-data.

Reply:
If you are moving the calls and referencing the new pacakge why do you still need the old package import?

Reply:
For this package it still needs `office-addin-cli` because it use the `parseNumber` function from it.